src/test/java/core/basesyntax/service/strategy/IncreaseOperationHandlerTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitRecordDto;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.exceptions.IncorrectAdditionalGoodsException;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.OperationType;
+import org.junit.Before;
+import org.junit.Test;
+
+public class IncreaseOperationHandlerTest {
+    private static OperationHandler increaseHandler;
+    private static FruitRecordDto correctFruitForIncrease;
+    private static FruitRecordDto incorrectFruitForIncrease;
+
+    @Before
+    public void setUp() {
+        FruitStorage.storage.clear();
+        increaseHandler = new IncreaseOperationHandler();
+        correctFruitForIncrease = new FruitRecordDto(OperationType.BALANCE,
+                new Fruit("fruit"), 21);
+        incorrectFruitForIncrease = new FruitRecordDto(OperationType.SUPPLY,
+                new Fruit("fruit"), -21);
+    }
```

Comment:
Same. You don't have to initialize increaseHandler before each test, you can do this one time before whole class into the method @BeforeClass